.github/workflows/test.yml

```
@@ -0,0 +1,64 @@
+name: Run linter and tests
+
+on:
+  push:
```

This allows the workflow to be triggered manually.

```suggestion
  workflow_dispatch:
  push:
```